controllers/operands/kubevirt.go

```diff
@@ -303,6 +310,30 @@ func NewKubeVirt(hc *hcov1beta1.HyperConverged, opts ...string) (*kubevirtcorev1
 	return kv, nil
 }
 
+func isAnnotationStateMeetingRequirements(hc *hcov1beta1.HyperConverged, annotations map[string]string) bool {
```

Comment:
Not sure about the name of this function. I find the name both a bit too long, and too generic.

Reply:
Are there alternative names you have in mind?